ci/docker/README.md

```diff
@@ -1,3 +1,6 @@
 # Build image with quiccir dependencies
+
+```bash
 docker build -f ci/docker/Dockerfile -t giacasti/ubuntu-llvm:22.04-17 .
-docker push giacasti/ubuntu-llvm:22.04-17
```

Comment:
We should not forget to put this into quicc docker repo when more or less stable

Reply:
added the image to the `quicc` docker repo